scripts/xdist/find_order_dependent_test_failures.py

```diff
@@ -179,7 +195,7 @@ def _create_and_check_test_files_for_failures(test_list, test_type):
         print('- test failures found.')
         return _get_pytest_command(output_file_path)
 
-    os.remove(temp_file.name)
+    # os.remove(temp_file.name)
```

Comment:
**Author Note:** I wanted to see the file with the list of tests to be run, but they were deleted. I think even with this commented out, these files are cleaned up at the start of each run? In any case, if we add a `--debug`` option, we could skip file removal when debugging, and just explain that in the help for `--debug``.